aws/ec2ids/ec2ids.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+)
+
+func stringInSlice(a string, list []string) bool {
+	for _, b := range list {
+		if b == a {
+			return true
+		}
+	}
+	return false
+}
+
+func main() {
+
+	// Command line arguments, which will be compared to the EC2
+	// instance tags values.
+	args := os.Args[1:]
+
+	sess, err := session.NewSession()
+	if err != nil {
+		panic(err)
+	}
+
+	// Create an EC2 service object in the "us-west-2" region
+	// Note that you can also configure your region globally by
+	// exporting the AWS_REGION environment variable
+	svc := ec2.New(sess, &aws.Config{Region: aws.String("eu-west-1")})
+
+	// Call the DescribeInstances Operation
+	resp, err := svc.DescribeInstances(nil)
```

Comment:
Possibly this needs further discussion. Might be better to describe a specific tag, rather than all instances.